2021/26xxx/CVE-2021-28689.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
Expected location (2021/28xxx/CVE-2021-28689.json) of file does not match actual location (2021/26xxx/CVE-2021-28689.json)

Reply:
fixed